designs/waralee/src/pocket.mjs

```diff
@@ -1,92 +1,77 @@
 import { pantsProto } from './pantsproto.mjs'
 
-function waraleePocket({
-  options,
-  measurements,
-  Point,
-  Path,
-  points,
-  paths,
-  Snippet,
-  snippets,
-  complete,
-  sa,
-  paperless,
-  macro,
-  part,
-}) {
-  if (false == options.frontPocket) {
-    return part
-  }
+export const pocket = {
+  name: 'waralee.pocket',
+  from: pantsProto,
+  draft: ({ options, Point, Path, points, paths, Snippet, snippets, sa, macro, store, part }) => {
+    if (false == options.frontPocket) {
+      return part.hide()
+    }
 
-  const c = 0.55191502449351
+    const c = 0.55191502449351
```

Comment:
I'm going to add this as a named export in core since it's something used often.